R/data.R

```diff
@@ -0,0 +1,88 @@
+#' Covariates/metadata of The Cancer Genome Atlas Liver Hepatocellular 
+#' Carcinoma gene expression data. 
+#' 
+#' This dataset is filtered to contain only sample with miRNA and mRNA gene
+#' expression profiles.  
+#' 
+#'
+#'
+#' @format A data frame with 417 rows and 73 variables.
+#' @source \url{https://doi.org/10.1016/j.cell.2017.05.046}
+"COV_TCGA_LIHC"
+
+
+#' Canonical pathways from Molecular Signatures Database, MsigDb V6.2
+#' 
+#'
+#' @format A list of 1329 pathways
+#' @source \urlhttp://www.gsea-msigdb.org/gsea/index.jsp}
```

Comment:
There's a missing `{`. It's a good idea to test building the docs after making changes with `devtools::document()`.